log/include/gz/utils/log/Logger.hh

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef GZ_UTILS_LOG_LOGGER_HH_
+#define GZ_UTILS_LOG_LOGGER_HH_
+
+#include <spdlog/spdlog.h>
+#include <memory>
+#include <string>
+#include <gz/utils/ImplPtr.hh>
+
+namespace gz::utils::log
+{
+  /// \brief Gazebo console and file logging class.
```

Comment:
Can you add a version (inline) namespace as most of our other libraries do?

Reply:
Added